docs/UserGuide.md

```diff
@@ -259,13 +236,36 @@ To achieve the best possible experience, avoid adding excessively long input to
 * The email address of your client
 * Should be in this format: `local-part@domain`
 * `local-part`: 
-  * should only contain *alphanumeric characters* and these *special characters* excluding parentheses (!#$%&'*+/=?{|}~^.-`).
+  * should only contain *alphanumeric characters* and these *special characters* excluding parentheses (!#$%&'*+/=?\{\}\|~^.-`).
   * should not start and end with special characters,
 * `domain`: 
   * must be at least **2** characters long. 
   * each domain label must start and end with alphanumeric characters
   * each domain label is separated using hyphens (if any)
 
+  
+#### `HELP_TOPIC`
```

Comment:
Is this suppose to be underscore or space. And is it just me but is the header not in bold?